xdp/src/tx_loop.rs

```diff
@@ -206,17 +201,6 @@ pub fn tx_loop<T: AsRef<[u8]>, A: AsRef<[SocketAddr]>>(
 
                     let mut skip = false;
 
-                    // sanity check that the address is routable through our NIC
```

Comment:
Why are we removing this check? should we not check this still if there is no tunnel?

Reply:
Thanks, this is clear!